src/pages/dashboard/listings/[slug]/submissions.tsx

```diff
@@ -21,25 +37,33 @@ interface Props {
   slug: string;
 }
 
+const selectedStyles = {
+  borderColor: 'brand.purple',
+};
+
 function BountySubmissions({ slug }: Props) {
   const router = useRouter();
   const { isOpen, onOpen, onClose } = useDisclosure();
   const { userInfo } = userStore();
   const [bounty, setBounty] = useState<Bounty | null>(null);
-  const [totalSubmissions, setTotalSubmissions] = useState(0);
-  const [totalWinners, setTotalWinners] = useState(0);
-  const [totalPaymentsMade, setTotalPaymentsMade] = useState(0);
+  const [totalSubmissions, setTotalSubmissions] = useState(-1);
```

Comment:
i dont remember doing these 0 -> -1, is this a auto merge error?

Reply:
no..

Reply:
change them back to 0